### ISSUE TITLE
GGM - DevTest - Go to HUD Screen # command not working in menu screens

### ISSUE THREAD

**Issue by Blood-Moon-Interactive** [TheGameCreators/GameGuruRepo#5856]
In the video linked below I'll explain that the button command for calling HUD screens does not work when used in a menu. I'm not sure if this intentional but in the use case I describe, a leaderboard screen (which is a HUD screen), there'd be no way to access it unless by pressing the designated Screen Toggle Key (in my case 'H'). 

https://youtu.be/SVchIDHTPc4

**Comment by LeeBamberTGC** [comment]
@Blood-Moon-Interactive @Kasseyus yes indeed, this was the intended behavior that this action only works when in the game and wanting to go from one HUD screen to another HUD screen.  The main menu screens are a hierarchy above and do more things like move from screens to levels to loading screens, etc. 

I have added warnings to help users know this, and changed this to a feature request in case we may want all kinds of things like starting the game while in the game, or playing a HUD before the game even starts :)

![Image](https://github.com/user-attachments/assets/1e71e345-ba27-45af-a11d-048d8b31fe6a)

![Image](https://github.com/user-attachments/assets/901db19b-e9a4-4bdc-b916-dec2d3f067fd)

**Comment by Blood-Moon-Interactive** [comment]
Ok thank you
